BaseMenu.cpp

```diff
@@ -991,7 +995,29 @@ static void UI_LoadBackgroundMapList( void )
 	EngFuncs::COM_FreeFile( afile );
 }
 
+static void UI_LoadSounds( void )
+{
+	memset( uiStatic.sounds, 0, sizeof( uiStatic.sounds ) );
+
+	if ( uiStatic.lowmemory )
+		return;
 
+	for ( int i = 0; i < SND_COUNT; i++ )
+	{
+		if ( !uiSounds[i] || *uiSounds[i] == '\0' )
+			continue;
+
+		if ( !EngFuncs::FileExists( uiSounds[i] ) )
+		{
+			size_t len = strlen( uiSoundOldPrefix );
+
+			if ( !strncmp( uiSounds[i], uiSoundOldPrefix, len ) )
+				sprintf( uiStatic.sounds[i], "%s%s", uiSoundNewPrefix, uiSounds[i] + len );
```

Comment:
Use `snprintf( uiStatic.sounds[i], sizeof( uiStatic.sounds[i] ), ...`

Reply:
Also writing `sounds/common/launch_upmenu1.wav` will overflow the 32 char array.